### PR TITLE
feat(access_key): change user_access table

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
resolve #15

### CODE REVIEW

**Comment by philloooo** [comment]
can you write a migration script to migrate existing data in old table to new table

**Comment by thanh-nguyen-dang** [comment]
yeah, I can do it

File: userdatamodel/user.py [hunk 1]
```diff
@@ -2,13 +2,16 @@
 import datetime
 from sqlalchemy import (
     Integer, String, Column, Table, Boolean, BigInteger, DateTime, text)
-from sqlalchemy import UniqueConstraint
+from sqlalchemy import UniqueConstraint, Index
 from sqlalchemy.orm.collections import attribute_mapped_collection
 from sqlalchemy.ext.associationproxy import association_proxy
-from sqlalchemy.dialects.postgres import ARRAY, JSONB
+from sqlalchemy.dialects.postgresql import ARRAY, JSONB
```
Comment (philloooo) [inline userdatamodel/user.py]: what's the difference between postgresql and postgres?

File: userdatamodel/user.py [hunk 2]
```diff
@@ -17,6 +20,20 @@
     Column('group_id', Integer, ForeignKey('research_group.id'))
 )
 
+class PrivilegeDict(MappedCollection):
```
Comment (philloooo) [inline userdatamodel/user.py]: can you add docs about what does this do?

Comment (thanh-nguyen-dang) [inline userdatamodel/user.py]: I added it

**Pull request merged**